### PR TITLE
ADEN-11851 Integrate native Triplelift

### PR DESCRIPTION
**Pull request by szymon-nawrocki**
### Ticket
https://fandom.atlassian.net/browse/ADEN-11851

### CODE REVIEW

**Review by dmnsobczak — APPROVED** [review]

File: platforms/ucp-desktop/bidders/prebid/triplelift.ts [hunk 1]
```diff
@@ -35,6 +35,11 @@ export function getTripleliftContext(): object {
 				],
 				inventoryCodes: ['Fandom_DT_BLB_728x90_hdx_prebid', 'Fandom_DT_BLB_970x250_hdx_prebid'],
 			},
+			ntv_ad: {
+				position: 'native',
+				sizes: [[728, 90]],
```
Comment (bckatarzyna) [inline platforms/ucp-desktop/bidders/prebid/triplelift.ts:40]: We probably don't need sizes here as it's a native ad. And they are not used in `prepareNativeConfig` code.

File: platforms/ucp-mobile/bidders/prebid/triplelift.ts [hunk 2]
```diff
@@ -17,6 +17,11 @@ export function getTripleliftContext(): object {
 				],
 				inventoryCodes: ['Fandom_MW_PF_300x250_hdx_prebid', 'Fandom_MW_PF_320x50_hdx_prebid'],
 			},
+			ntv_ad: {
+				position: 'native',
+				sizes: [[320, 50]],
```
Comment (bckatarzyna) [inline platforms/ucp-mobile/bidders/prebid/triplelift.ts:22]: We probably don't need sizes here as it's a native ad. And they are not used in `prepareNativeConfig` code.

File: src/ad-bidders/prebid/adapters/triplelift.ts [hunk 3]
```diff
@@ -31,4 +49,48 @@ export class Triplelift extends PrebidAdapter {
...
 	}
+
+	prepareNativeConfig(
+		template: string,
+		code,
+		{ inventoryCodes }: PrebidAdSlotConfig,
+	): PrebidAdUnit {
+		return {
+			code,
+			mediaTypes: {
```
Comment (bckatarzyna) [inline src/ad-bidders/prebid/adapters/triplelift.ts:60]: This part (lines 60-85) could be extracted to a separate function (`getNativeMediaTypes`?) in a separate file as it's used in both appnexus and triplelift integration. And probably will be used in all next native integrations

Comment (szymon-nawrocki) [reply src/ad-bidders/prebid/adapters/triplelift.ts:60]: Great idea! Done ✅

**Review by bckatarzyna — APPROVED** [review]

File: platforms/ucp-desktop/bidders/prebid/triplelift.ts [hunk 4]
```diff
@@ -35,6 +35,10 @@ export function getTripleliftContext(): object {
 				],
 				inventoryCodes: ['Fandom_DT_BLB_728x90_hdx_prebid', 'Fandom_DT_BLB_970x250_hdx_prebid'],
 			},
+			ntv_ad: {
+				position: 'native',
+				inventoryCodes: ['Fandom_DT_NativeIC_728x90'],
+			},
```
Comment (bckatarzyna) [inline platforms/ucp-desktop/bidders/prebid/triplelift.ts:41]: This change should be removed as the config is now in `triplelift-native` (to not duplicate requests)

File: platforms/ucp-mobile/bidders/prebid/triplelift.ts [hunk 5]
```diff
@@ -17,6 +17,10 @@ export function getTripleliftContext(): object {
 				],
 				inventoryCodes: ['Fandom_MW_PF_300x250_hdx_prebid', 'Fandom_MW_PF_320x50_hdx_prebid'],
 			},
+			ntv_ad: {
+				position: 'native',
+				inventoryCodes: ['Fandom_Mobile_NativeIC_320x50'],
+			},
```
Comment (bckatarzyna) [inline platforms/ucp-mobile/bidders/prebid/triplelift.ts:23]: This change should be removed as the config is now in `triplelift-native` (to not duplicate requests)

**Pull request closed without merging**